include/tvm/relay/pass/alpha_eq.h

```diff
@@ -0,0 +1,53 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file tvm/relay/pass/alpha_eq.h
```

Comment:
move to relay/pass/pass.h

Reply:
There are two kinds of reasoning here. Either we give each pass one file, and that avoids certain recompile if there is no extra dependency on that pass. Or we put all the pass into one file for now so it is easy for the user to lookup